docs/deploy-cluster.md

```diff
@@ -0,0 +1,144 @@
+---
+
+This tutorial provides a introduction to deploy `Nebula` cluster.
+
+---
+
+#### Download and install package
+
+First at all, you can download rpm or deb from [Here]().
+
+For `CentOS`:
+
+```
+rpm -ivh nebula-{VERSION}-0.el7.centos.x86_64.rpm
+```
+
+For `Ubuntu`:
+
+```
+dpkg -i nebula-{VERSION}-0.el7.centos.x86_64.deb
+```
+
+By default, the config files are under `/usr/local/etc`, you should modify the `meta_server_addrs` to set the Meta Server's address. 
+
+In order to enable multi copy Meta service, you should set the meta addresses split by comma into `meta_server_addrs`.
+
+Using `data_path` to set `Meta` and `Storage`'s underlying storage directory.
```

Comment:
you can find an example as xxx.default

Reply:
I have put the config's key and default value after that.